### DISCUSSION TITLE
Looking for a proper issue to get started

### DISCUSSION THREAD

**Discussion by arousta** [keepassxreboot/keepassxc#8906] [Q&A] [answered]
Hi, I'm using keepassxc for a while and like it. Want to start contributing to the project. Looked at the "Contributing" doc which says filter for `begginer` and `help-wanted` issues. But the former has 0 and the latter has only 1 issue for Mac currently. To give some info on my background: I have +6 years of experience with C++98, have been following the updates to the standard since C++11, but never got the opportunity to do serious work as busy with other stuff. Also have not used Qt so far. Is there any issue which touches the right amount of concepts (given my backgroud)? Much appreciated and looking forward to contributing more and more as I learn :)

**Comment by droidmonkey** [comment, marked as answer]
Check out the `hacktoberfest` label. Welcome to the fun!

**Comment by arousta** [comment]
Super! very excited to contribute and learn new stuff! :+1:
